1_fetch/src/create_grids.R

```diff
@@ -66,3 +66,26 @@ subset_grids_to_aoi <- function(grid, aoi_poly, dist_m){
   
 }
 
+
+
+#' Return bounding box as a vector
+#' 
+#' @description Function to find the bounding box for an sf object and return
+#' as a vector
```

Comment:
I think maybe a cleaner way to do this (noting that we might be able to still get the mapping over polygons working again), would be to map over a list where the list name was the id and the list values were the bbox vectors. I think that would make extracting the info cleaner in `inventory_wqp()`, too.

```r
library(sf)
library(dplyr)

grid <- tar_read(p1_conus_grid_aoi)

# Setup for mapping
p1_conus_grid_aoi_bbox <- lapply(sf::st_geometry(grid), sf::st_bbox) %>% 
  setNames(grid$id)

# Use in mapping scenario, the input is already a vector of the bbox
# ready to subset with `bbox$xmin`
purrr::map(p1_conus_grid_aoi_bbox, function(bbox_i) {
  # The bbox is already un-tabled/unlisted in vector form!
  message(bbox_i)
  return(bbox_i$xmin)
})
```

Reply:
Ooh, great suggestion - this is definitely cleaner. Even though I ultimately expect to map over the polygons instead of these bbox's, I've made this change in [dfeb379](https://github.com/USGS-R/ds-pipelines-targets-example-wqp/pull/21/commits/dfeb379fd350034401733baa4cc351e1e2dab16a).